### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by ZacSweers**
Gradle 6.8-rc-1, misc dependencies, kotlin/dokka 1.4.20

### CODE REVIEW

**Comment by ZacSweers** [comment]
Ignore the `pull_request_target` checks, I'm removing that in this PR as it doesn't work the way we thought

File: keeper-gradle-plugin/gradle.properties [hunk 1]
```diff
@@ -14,13 +14,16 @@
...
+org.gradle.jvmargs=-XX:MaxMetaspaceSize=512m
+
 kapt.includeCompileClasspath=false
 
 POM_NAME=Keeper
 POM_ARTIFACT_ID=keeper
 POM_PACKAGING=jar
 GROUP=com.slack.keeper
-VERSION_NAME=0.8.0-SNAPSHOT
+VERSION_NAME=0.8.0-LOCAL1
```
Comment (jschear) [inline keeper-gradle-plugin/gradle.properties:26]: Intentional change?

File: keeper-gradle-plugin/src/main/java/com/slack/keeper/KeeperPlugin.kt [hunk 2]
```diff
@@ -122,11 +122,15 @@ public class KeeperPlugin : Plugin<Project> {
             property("android.enableR8")?.toString()?.toBoolean() != false
         if (r8Enabled) {
           appExtension.onApplicableVariants(project, extension) { testVariant, appVariant ->
+            val appR8Task = "minify${appVariant.name.capitalize(Locale.US)}WithR8"
+            val androidTestL8Task = "l8DexDesugarLib${testVariant.name.capitalize(Locale.US)}"
+            logger.debug("KEEPER: Patching $androidTestL8Task with ")
```
Comment (jschear) [inline keeper-gradle-plugin/src/main/java/com/slack/keeper/KeeperPlugin.kt:127]: nit: remove this or add `appR8Task` to it?

File: keeper-gradle-plugin/gradle.properties (hunk 1 above)
Comment (ZacSweers) [reply keeper-gradle-plugin/gradle.properties:26]: errrrr nope will fix

File: keeper-gradle-plugin/src/main/java/com/slack/keeper/KeeperPlugin.kt (hunk 2 above)
Comment (ZacSweers) [reply keeper-gradle-plugin/src/main/java/com/slack/keeper/KeeperPlugin.kt:127]: I think this was left over from some debugging... let me look into it more 😓

**Comment by ZacSweers** [comment]
Proceeding with merge as the non-`pull_request_target` checks are passing and the above two issues are resolved now

image

**Pull request merged**